web-client/iron-remote-desktop/src/services/wasm-bridge.service.ts

```diff
@@ -426,12 +442,14 @@ export class WasmBridgeService {
         const syncScrollLockActive = evt.getModifierState(LockKey.SCROLL_LOCK);
         const syncKanaModeActive = evt.getModifierState(LockKey.KANA_MODE);
 
-        this.session?.synchronize_lock_keys(
-            syncScrollLockActive,
-            syncNumsLockActive,
-            syncCapsLockActive,
-            syncKanaModeActive,
-        );
+        this.session?.extension_call({
+            SynchronizeLockKeys: {
+                scroll_lock: syncScrollLockActive,
+                num_lock: syncNumsLockActive,
+                caps_lock: syncCapsLockActive,
+                kana_lock: syncKanaModeActive,
+            },
+        });
```

Comment:
thought: Maybe it’s simpler to keep `synchronize_lock_keys` in the common API, the same way as previously. It’s fine if it’s a no-op in `ironvnc-web`.

Reply:
Reverted